src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    private static final String WORD_BUY = "buy";
+    private static final String WORD_SUPPLY = "supply";
+    private static final String COMMA = ",";
+    private static final int NUMBER_ZERO = 0;
+    private static final String SPACE = " ";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method should contain three calls of other methods: readFile() - returns content, generateReport() -(you pass contents here), writeToFile()-(you pass report and filename here ), this will be a good code decomposition. example:
        String data = readFile(fromFileName);
        String report = generateReport(data);
        writeToFile(toFileName, report);